contrib/authors_update.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# See file LICENSE for terms.
+#
+
+set -eu
+set -o pipefail
+
+range="${1?Provide commit range like orig/v1.17..orig/v1.18}"
+
+if [ ! -f AUTHORS ]
+then
+    echo "AUTHORS file not accessible"
+    exit 1
+fi
+
+# Failure triggered if range is not valid
+git --no-pager log --oneline --pretty=format:"%h %an <%ae> %s" "$range"
```

Comment:
maybe add a log if the range is not valid?

Reply:
I let git output provide detailed:
```
/contrib/authors_update.sh bad..range
fatal: ambiguous argument 'bad..range': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```